application/minedtx/mined_test.go

```diff
@@ -134,6 +134,49 @@ func makeTxInitial(t *testing.T, ownerSigner objs.Signer) (objs.Vout, *objs.Tx)
 	return consumedUTXOs, tx
 }
 
+func makeTxInitial2(t *testing.T, ownerSigner objs.Signer, numTxIns uint32) (objs.Vout, *objs.Tx) {
```

Comment:
How does this differ from `makeTxInitial`?

Reply:
It allows for specifying the number of TxIns. I will use this version and remove the other.